kalite/topic_tools/__init__.py

```diff
@@ -165,7 +165,7 @@ def get_exercise_cache(force=False, language=settings.LANGUAGE_CODE):
                 exercise_template = os.path.join(exercise_lang, exercise_file)
 
 
-            with i18n.translate_block(exercise_lang):
+            with i18n.translate_block(language):
```

Comment:
Looking at our central server's `supported_languages.json` we don't have these cases yet, so this regression is not a blocker for me.